src/PHPCR/Shell/Resources/config.dist/alias.yml

```diff
@@ -49,7 +49,6 @@ checkin: version:checkin
 ci: version:checkin
 co: version:checkout
 checkout: version:checkout
-cp: version:checkpoint
 checkpoint: version:checkpoint
```

Comment:
fine with dropping the cp line as its already copy. having checkpoint as alias seems good enough to me.